stable/portus/templates/portus-post-delete-job.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.portus.cleanup.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ template "portus.fullname" . }}-delete-jobs"
+  labels:
+    app: "{{ template "portus.name" . }}"
+    chart: {{ template "portus.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: portus
+  annotations:
+    "helm.sh/hook": post-delete
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app: {{ template "portus.name" . }}
```

Comment:
Add `release` label to selector.

Reply:
Thanks for the reminder.
I will get that fixed.